recipe_scrapers/schoolofwok.py

```diff
@@ -0,0 +1,172 @@
+from collections import defaultdict
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import IngredientGroup
+from ._utils import normalize_string
+
+
+class SchoolOfWok(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "schoolofwok.co.uk"
+
+    def author(self):
+        return "School of Wok"
+
+    def site_name(self):
+        return "School of Wok"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def cuisine(self):
+        categoryheader = self.soup.find(
+            string=lambda text: text and text.lower() == "cuisine"
+        )
+
+        if not categoryheader:
+            return ""
+
+        return categoryheader.find_next("p").get_text()
```

Comment:
```suggestion
        return categoryheader.find_next("p").get_text() if categoryheader else None
```
This simplifies the code a little bit and returns `None` rather than `""`